cmd/minikube/cmd/config/validations.go

```diff
@@ -50,6 +51,14 @@ func IsValidDiskSize(name string, disksize string) error {
 	return nil
 }
 
+func IsValidURL(name string, location string) error {
+	_, err := url.Parse(location)
```

Comment:
Is this the desired behavior?  You can check the scheme of the parsed url Object.  This would at least not allow for "asdkhfajdfhas" (this shouldn't pass right?)